internal/pkg/aws/codestar/codestar_test.go

```diff
@@ -27,6 +27,11 @@ func TestCodestar_WaitUntilStatusAvailable(t *testing.T) {
 		connection := &CodeStar{}
 		connectionARN := "mockConnectionARN"
 
+		ctrl := gomock.NewController(t)
+		defer ctrl.Finish()
+		m := mocks.NewMockapi(ctrl)
+		m.EXPECT().GetConnection(gomock.Any()).AnyTimes()
```

Comment:
huh does this work without a `Return`?

Reply:
I wonder what would `aws.StringValue(output.Connection.ConnectionStatus)` be without a `Return` 🤔

Reply:
🤔 I just looked into the gomock library they return `nil` by default for each return value! That's probably why it was working haha